### ISSUE TITLE
Generate report for debugging

### ISSUE THREAD

**Issue by lyie28** [lavague-ai/LaVague#377]
Create option to create report for debugging with key info from logs for easier debugging

**Comment by paulpalmieri** [comment]
Yes please! This would be a really good addition. I believe logs should be saved to a local file by default and contain every loggable information even in case of interrupt.
